lang/cs/Org.Apache.REEF/Org.Apache.REEF.nuspec

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0"?>
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+<package >
+  <metadata>
+    <id>Org.Apache.REEF</id>
+    <version>$version$</version>
+    <title>Org.Apache.REEF</title>
+    <authors>The Apache REEF project</authors>
+    <owners>The Apache REEF project</owners>
+    <licenseUrl>http://www.apache.org/licenses/LICENSE-2.0</licenseUrl>
+    <projectUrl>http://reef.apache.org/</projectUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>Provide single entry point that contains all necessary REEF/WAKE/TANG projects.</description>
+    <copyright>The Apache Software Foundation</copyright>
+    <tags>Single entry point to reference all Reef/Wake/Tang projects </tags>
+    <dependencies>
+      <dependency id="Microsoft.Hadoop.Avro" version="1.5.6" />
+      <dependency id="Newtonsoft.Json" version="8.0.3" />
+      <dependency id="Org.Apache.REEF.Client" version="$version$" />
+      <dependency id="Org.Apache.REEF.Common" version="$version$" />
+      <dependency id="Org.Apache.REEF.Driver" version="$version$" />
+      <dependency id="Org.Apache.REEF.Evaluator" version="$version$" />
+      <dependency id="Org.Apache.REEF.Examples" version="$version$" />
+      <dependency id="Org.Apache.REEF.IMRU" version="$version$" />
+      <dependency id="Org.Apache.REEF.IO" version="$version$" />
+      <dependency id="Org.Apache.REEF.Network" version="$version$" />
+      <dependency id="Org.Apache.REEF.Tang" version="$version$" />
+      <dependency id="Org.Apache.REEF.Utilities" version="$version$" />
+      <dependency id="Org.Apache.REEF.Wake" version="$version$" />
+      <dependency id="protobuf-net" version="2.0.0.668" />
+      <dependency id="Rx-Core" version="2.2.5" />
+    </dependencies>
+  </metadata>
+  <files>
+    <file src="..\bin\$Platform$\$Configuration$\Org.Apache.REEF.All\Org.Apache.REEF.All.dll" target="lib\net45" />
+    <file src="..\bin\$Platform$\$Configuration$\Org.Apache.REEF.All\Org.Apache.REEF.All.pdb" target="lib\net45" />
+    <file src="..\bin\$Platform$\$Configuration$\Org.Apache.REEF.Client\Org.Apache.REEF.Client.dll" target="lib\net45" />
```

Comment:
Try to reference all the projects needed that would ensure those dlls are available in the bin folder of this project after build. Then change the reference here to pick up those dlls from the current project bin folder ..\bin\$Platform$\$Configuration$\Org.Apache.REEF\...  . That would make this project self contained.